### PR TITLE
Refactor class javascript

### PR DESCRIPTION
**Pull request by beyk**
Update Html and Css
Added updated JS 
- Loading content on real-time without page refresh now
- Added classes to fulfill functionalities

### CODE REVIEW

File: script/main.js [hunk 1]
```diff
@@ -1,50 +1,112 @@
-class Books {
-  constructor(title = '', author = '') {
-    this.title = title;
-    this.author = author;
-  }
-}
+import { Books, foo } from './classes.js';
 
 let bookdata = [];
```
Comment (zieeco) [inline script/main.js]: - Hey @Team :wave: great job using classes, however according to the project requirement, you should only use object and array inside the classes and not outside the classes. as you've used the array outside the class. Let's refactor the code to meet the project requirement as much as we can:

*project requirement*
Screenshot from 2023-04-26 12-02-22

File: script/main.js [hunk 2]
```diff
@@ -1,50 +1,112 @@
...
 document.addEventListener('DOMContentLoaded', () => {
   window.onload = () => {
     if (localStorage) {
-      const localStorageItem = localStorage.getItem('bookdata');
+      const localStorageItem = localStorage.getItem('booklist');
       bookdata = JSON.parse(localStorageItem);
       if (bookdata == null) {
         bookdata = [];
       }
+      booklist.loadbooks(bookdata);
```
Comment (zieeco) [inline script/main.js]: - The same issue I pointed out above also applies to this.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Team,




Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Descriptive PR. ✔️
- Good commit messages :+1: 
- Classes used :+1: 

## Required Changes ♻️
_Check the comments under the review._




## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script/main.js (hunk 2 above)
Comment (misalazeem) [reply script/main.js]: @zieeco  just parsing the data from localstorage to class. Are we not allowed to do that?

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @misalazeem, and @beyk👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- Highly organized code. ✔️
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- No linter errors. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**